src/rabbit_federation_mgmt.erl

```diff
@@ -26,17 +26,20 @@
 
 -include_lib("rabbitmq_management_agent/include/rabbit_mgmt_records.hrl").
 
-dispatcher() -> [{"/federation-links",        ?MODULE, []},
-                 {"/federation-links/:vhost", ?MODULE, []}].
+dispatcher() -> [{"/federation-links",        ?MODULE, [all]},
+                 {"/federation-links/:vhost", ?MODULE, [all]},
+                 {"/federation-down-links",        ?MODULE, [down]},
```

Comment:
I'd use `/federation-links/down` and `/federation-links/:vhost/down`. I understand that `/federation-links/down` conflicts with `/federation-links/:vhost` but if we put it higher on the list, shouldn't it take priority?